App.js

```diff
@@ -281,8 +288,9 @@ class App extends React.Component<Props, *> {
                 }}
               />}
               {!!getEnv().SHOW_LANG_TOGGLE && <Button
-                title={`Change lang (current: ${i18n.language})`} // eslint-disable-line i18next/no-literal-string
-                onPress={() => changeLanguage(i18n.language === DEV_LANG ? DEFAULT_LANGUAGE : DEV_LANG)}
+                title={`Change lang (current: ${i18next.language})`} // eslint-disable-line i18next/no-literal-string
+                // eslint-disable-next-line i18next/no-literal-string
+                onPress={() => changeLanguage(i18next.language === 'lt' ? localeConfig.defaultLanguage : 'lt')}
```

Comment:
Dev language will be `lt` now?  🇱🇹 👀

Reply:
Just for now. As mentioned in description ("N.B" part), I've added lt as supported language for testing. Will revert those changes after approval.